packages/cashier/src/components/recent-transaction/recent-transaction.tsx

```diff
@@ -23,31 +28,22 @@ const RecentTransaction = observer(() => {
     };
 
     const SideNoteContainer = ({ children }: React.PropsWithChildren<unknown>) => (
-        <div className='recent-transaction__wrapper'>
+        <div className={`recent-transaction__wrapper ${is_wallet ? 'recent-transaction__wrapper--wallet' : ''}`}>
```

Comment:
@hamid-deriv 
what do you think about using classNames library here?
IMHO it's more readable than using interpolation
```suggestion
        <div className=classNames('recent-transaction__wrapper', {
        'recent-transaction__wrapper--wallet': is_wallet
        })>
```
just suggestion, you can ignore it